HowToFIDO.md

```diff
@@ -270,26 +273,26 @@ Reauthentication might happen for the following reasons:
     re-confirm control over the user session.
 
 Let’s look at the last case first: when it’s time to re-authenticate for
-a sensitive action, check whether you have a credential id for this user
-*for the purpose of reauthentication*, i.e., the kind of credential id
-obtained from [<span class="underline">opting the user into FIDO-based
-reauthentication</span>](#opting-into-fido-based-reauthentication). Make
-sure it’s associated with the user *and* device - for example, check a
-cookie or read from local storage.
-
-If *no credential id is available*, serve a traditional login challenge
+a sensitive action, check whether you have a `2FACapableFIDOCredsAvailable` flag for this user
+*for the purpose of reauthentication*.
```

Comment:
What is the conditional requirement around the words: *for the purpose of reauthentication*? Either the `2FACapableFIDOCredsAvailable` flag is sufficient to pivot to what happens next, or it's not. I think these words should be deleted.